packages/tooltip/src/Tooltip.tsx

```diff
@@ -211,7 +227,7 @@ function Tooltip({
       onClick: (e: MouseEvent) => {
         // ensure that we don't close the tooltip when content inside tooltip is clicked
         if (e.target !== tooltipRef.current) {
-          setOpen((curr: boolean) => !curr);
+          setOpen((curr: boolean) => enabled && !curr);
```

Comment:
Should we check for enabled on the line above, rather than within the setOpen function?

Reply:
I put it here because in this case, it will close it if it's disabled. If enabled is `false` and the tooltip is open for some reason, we want the tooltip to close.